docs/en/advanced_tutorials/initialize.md

```diff
@@ -1,3 +1,325 @@
 # Initialization
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/initialize.html).
+Usually, we'll customize our module based on [nn.Module](https://pytorch.org/docs/stable/nn.html?highlight=nn%20module#module-torch.nn.modules), which is implemented by Native PyTorch. Also, [torch.nn.init](https://pytorch.org/docs/stable/nn.init.html?highlight=kaiming#torch.nn.init.kaiming_normal_) could help us initialize the parameters of the model easily. To simplify the process of model construction and initializationm, MMEngine design the [BaseModule](mmengine.model.BaseModule) to help us define and initialize the model from config easily.
+
+## Initialize the model from config
+
+The core function of `BaseModule` is that it could help us to initialize the model from config. Subclasses inherit from `BaseModule` could define the `init_cfg` in the `__init__` function, and we can choose the method of initialization by configuring `init_cfg`.
```

Comment:
```suggestion
The core function of `BaseModule` is that it could help us to initialize the model from config. Subclasses inherited from `BaseModule` could define the `init_cfg` in the `__init__` function, and we can choose the method of initialization by configuring `init_cfg`.
```